tests/test_models.py

```diff
@@ -54,9 +54,26 @@
 ]
 
 
-# Test HuggingFace Models (GPT-2)
+def test_modalities():
+    # dismiss sequences that are too long for our test checkpoint
+    test_cases = LOGLIKELIHOOD_TEST_CASES[:5]
+    modalities = models.get_model("modalities").create_from_arg_string("pretrained='testdata/models/modalities/checkpoint'")
+    results = modalities.loglikelihood(test_cases)
+    for loglikelihood, is_max_loglikelihood in results:
+        assert type(loglikelihood) == float
+        assert type(is_max_loglikelihood) == bool
+
+    # test empty context
+    modalities.loglikelihood([("", "test")])
+    greedy_len = 20
+    (gen,) = modalities.greedy_until(
+        [("The quick brown fox jumps over the lazy", {"until": [".", "\n"], "max_length": greedy_len})]
+    )
```

Comment:
The `(gen,)` simply accesses the first element of the return, right?  I would rewrite it for more clarity.

```suggestion
    gen = modalities.greedy_until(
        [("The quick brown fox jumps over the lazy", {"until": [".", "\n"], "max_length": greedy_len})]
    )[0]
```